common/unsafe/src/main/java/org/apache/spark/unsafe/types/CalendarInterval.java

```diff
@@ -69,7 +70,7 @@ private static long toLong(String s) {
 
   /**
    * Convert a string to CalendarInterval. Return null if the input string is not a valid interval.
-   * This method is case-sensitive and all characters in the input string should be in lower case.
+   * This method is case-insensitive.
```

Comment:
This change the description without changing the function body. This means the previous function descriptions wrong. However, for example, the function body still has case-sensitive function like `s.equals("interval")`. For me, the previous function description is correct and new one is incorrect. Let's recover this comment change.